src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -87,6 +87,16 @@ public sealed class SystemConformanceProvider
             _urlResolver = urlResolver;
             _contextAccessor = contextAccessor;
             _searchParameterStatusManager = searchParameterStatusManager;
+
+            if (!string.IsNullOrEmpty(_configuration.Value.Versioning.Default))
+            {
+                _logger.LogInformation("Default version is:{VersioningDefault}.", _configuration.Value.Versioning.Default);
+
+                foreach (var resourcetype in _configuration.Value.Versioning.ResourceTypeOverrides)
+                {
+                    _logger.LogInformation("{ResourceTypeKey} version overridden to:{ResourceTypeValue}.", resourcetype.Key, resourcetype.Value);
```

Comment:
nit: maybe change `ResourceTypeValue` to `ResourceTypeVersioningOverride`?

Reply:
above changes done